app/store/ephemeral_store.js

```diff
@@ -19,6 +19,11 @@ class EphemeralStore {
 
     getNavigationTopComponentId = () => this.navigationComponentIdStack[0];
 
+    clearNatigationComponents = () => {
```

Comment:
Perhaps you meant `clearNavigationComponents`. If so, based on the spelling of the function, this may not be getting used and could be removed.